libs/deps/mas/commerce.js

```diff
@@ -1,4 +1,4 @@
 //# sourceMappingURL=commerce.js.map
```

Comment:
I know there's a ticket to handle this already, but sourcemaps references have to go (if the sourcemap is not available).

Reply:
with https://github.com/adobecom/milo/pull/2798 this should be fixed.